opendata-assets/src/scss/drupal/component/form.scss

```diff
@@ -32,11 +30,13 @@
 
     // Reset rounded corners
     &:first-child .form-control:first-child {
-      @include border-left-radius($border-radius-base);
+      border-top-left-radius: $border-radius-base;
+      border-bottom-left-radius: $border-radius-base;
     }
 
     &:last-child .form-control:first-child {
-      @include border-right-radius($border-radius-base);
+      border-top-right-radius: $border-radius-base;
+      border-bottom-right-radius: $border-radius-base;
     }
```

Comment:
Could be done using `border-start-radius` and `border-end-radius` mixins instead.